mautic/mautic
# Fix tracking for unsubscribe landing page

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

The landing page used as the preferences page does not track contact visits. This PR fixed it



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a landing page as a preferences center
3. Send email with set preferences center page for email
4. After receiving the email go to unsubscribe page
5. After PR you should see visits stats on landing pages and hits are increased

![image](https://github.com/mautic/mautic/assets/462477/a0babc2a-9548-4277-9eea-c354aa3f64e5)
